CONTRIBUTING.adoc

```diff
@@ -156,17 +188,30 @@ NOTE: This is a notice that you should pay attention to!
 CAUTION: This is a common mistake!
 
 
-There are too many other helpful macros and formatting options to list here, so it
-is recommended that you refer to the link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference[quick reference]
+There are too many other helpful macros and formatting options to list here, so
+it is recommended that you refer to the
+link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference[quick reference]
 to become more familiar with what is available.
 
 
+==== Adding a Tutorial page
+
+A tutorial is presented on its own page, each of which is written up as an
+`.adoc` file located in the link:content/doc/tutorials[`content/doc/tutorials/`]
+directory. If an `.adoc` file name begins with a underscore (e.g.
+link:content/doc/tutorials/_prerequisites.adoc[`content/doc/tutorials/_prerequisites.adoc`]),
+this means that the content is used as an
+link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#include-files[Asciidoc
+inclusion] on another page.
+
+
 ==== Adding a Handbook chapter
 
-The different chapters for the Handbook are located in the `content/doc/book/` directory
-and generally speaking each `.adoc` file represents a single chapter of the book.
+The different chapters for the Handbook are located in the
+link:content/doc/book[`content/doc/book/`] directory and
+generally speaking each `.adoc` file represents a single chapter of the book.
 Chapters are automatically surfaced on the Handbook home page (provided by
-`content/doc/book.html.haml`).
+link:content/doc/book/index.html.haml[`content/doc/book/index.html.haml`]).
```

Comment:
Again the github url, I think.

Reply:
No worries - I'll alter this based on the decision above.